Gruntfile.js

```diff
@@ -120,10 +143,16 @@ module.exports = function (grunt) {
         'watch'
     ]);
 
+    grunt.registerTask('lint', 'Run the JS linters.', [
+        'eslint',
+        'jscs'
+    ]);
+
     grunt.registerTask('test', [
         'dist',
         'test-module-isolation',
-        'karma:ci'
+        'karma:ci',
+        'lint'
```

Comment:
We usually lint before doing anything else, so I think this should go as the very first task